### PR TITLE
blinkR30 examples for PRU BBAI

### PR DESCRIPTION
**Pull request by lorforlinux**
@jadonk @MarkAYoder please review my code :) Everything has been tested on BBAI.

These PRU examples are now as simple as we have on BBB/BBBWL ;)

### CODE REVIEW

File: BeagleBone/AI/pru/blinkR30.pru1_0.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
+////////////////////////////////////////
+//	blinkR30.c
+//	Blinks LEDs wired to pr1_pru0 pins by writing register R30 on the PRU
+//	Wiring:	P9_26 connects to the plus lead of an LED.  The negative lead of the
+//			LED goes to a 220 Ohm resistor.  The other lead of the resistor goes
+//			to ground.
+//	Setup:	None
+//	See:	cloud9/common/prugpio.h for which pins attach to pr1_pru0.
```
Comment (jadonk) [inline BeagleBone/AI/pru/blinkR30.pru1_0.c:8]: Do we need 'cloud9' here in the path?

File: BeagleBone/AI/pru/blinkR30.pru1_1.c [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ////////////////////////////////////////
 //	blinkR30.c
-//	Blinks LEDs wired to P9_25 (and others) by writing register R30 on the PRU
+//	Blinks LEDs wired to pr1_pru1 pins by writing register R30 on the PRU
 //	Wiring:	P9_16 connects to the plus lead of an LED.  The negative lead of the
 //			LED goes to a 220 Ohm resistor.  The other lead of the resistor goes
 //			to ground.
 //	Setup:	None
-//	See:	prugpio.h to see which pins attach to R30
+//	See:	cloud9/common/prugpio.h for which pins attach to pr1_pru1.
```
Comment (jadonk) [inline BeagleBone/AI/pru/blinkR30.pru1_1.c:8]: I think the "cloud9" in the path is a bit confusing.

**Review by jadonk — APPROVED** [review]

File: BeagleBone/AI/pru/blinkR30.pru1_1.c (hunk 2 above)
Comment (lorforlinux) [reply BeagleBone/AI/pru/blinkR30.pru1_1.c:8]: It was really helpful for me so i added it, should i remove it in next PR?

**Comment by lorforlinux** [comment]
@jadonk please add the hacktoberfest label on this PR also :)

**Pull request merged**